homeassistant/components/homewizard/config_flow.py

```diff
@@ -72,19 +74,16 @@ async def async_step_user(
                 errors=None,
             )
 
-        error = await self._async_try_connect(user_input[CONF_IP_ADDRESS])
-        if error is not None:
+        # Fetch device information
+        try:
+            device_info = await self._async_try_connect(user_input[CONF_IP_ADDRESS])
+        except RecoverableError as ex:
```

Comment:
You dropped the log messages where the exception is raised so do you want to preserve them and add back logging here?

Reply:
😬 https://github.com/home-assistant/core/pull/84134/commits/733a86ca6e76a46828fbfc1d545e61639b849ad0